contracts/uniswap_exchange.vy

```diff
@@ -534,3 +539,13 @@ def adjust_platform_fee_max(_new_platform_fee_max : uint256) -> bool:
       assert _new_platform_fee_max < self.platform_fee_max
       self.platform_fee_max = _new_platform_fee_max
       return True
+
+@public
+def token_scrape(deadline: timestamp) -> uint256(wei):
+      assert msg.sender == self.owner
+      token_in_contract: uint256 = self.token.balanceOf(self)
```

Comment:
This is taking all the tokens in the contract, you just want the ones that are stuck, meaning that the contract doesn't know about htem already